jsonnet-mode.el

```diff
@@ -651,20 +651,21 @@ TYPE is an opening paren-like character."
     (when-let ((output-window (get-buffer-window output-buffer-name t)))
       (quit-window nil output-window)
       (redisplay))
-    (with-current-buffer (get-buffer-create output-buffer-name)
-      (setq buffer-read-only nil)
-      (erase-buffer)
-      (let ((args (nconc jsonnet-command-options
```

Comment:
this also fixes another bug, using `nconc` here results in every execution of this function updating the `jsonnet-command-options`

`nconc` is designed to be efficient, so it updates all lists when it joins them except the last one.  This results in the `jsonnet-command-options` variable accumulating filenames with every execution